src/Security/Core/Authorization/ExpressionLanguageProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Security\Core\Authorization;
+
+use Symfony\Component\ExpressionLanguage\ExpressionFunction;
+use Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface;
+
+/**
+ * Adding ApiPlatform specific Expression Language functions.
+ *
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
+ */
+class ExpressionLanguageProvider implements ExpressionFunctionProviderInterface
+{
+    public function getFunctions()
+    {
+        return [
+            new ExpressionFunction('is_granted', function ($attributes, $object = 'null') {
```

Comment:
Wouldn't it conflict with the one registered by SensioFrameworkExtraBundle?

Reply:
One would just override the other (depending on the order they are called). And as far as I could see the idea was to provide the same logic, right? I mean, we could only register it if the bundle is not loaded if that makes you more comfortable? Or just tell the users to require the framework-extra-bundle if they want this function?

Reply:
I just want to be sure it works as intended in all cases :)

Reply:
It should :)